src/popup.tsx

```diff
@@ -29,31 +29,56 @@ const env = buildEnvironment(chromeApiSingleton);
 const core = new Core(env);
 core.load().catch(console.error);
 
+let globalCssTemplates = `
+  @import url("https://fonts.googleapis.com/css?family=Roboto");
+
+  body {
+    background: #f6f8fc;
+    color: #444;
+    margin: 0 auto;
+    padding: 8px;
+    width: 600px;
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+  }
+
+  a {
+    color: #000;
+  }
+
+  .nav-tabs {
+    border-bottom: none;
+  }
+`;
+
+// Temporary workaround for secondary monitors on MacOS where redraws don't happen
+// TODO: Remove once Chromium fixes bug.
+// Ref: https://stackoverflow.com/questions/56500742/why-is-my-google-chrome-extensions-popup-ui-laggy-on-external-monitors-but-not/64113061#64113061
+// @See https://bugs.chromium.org/p/chromium/issues/detail?id=971701
+const isSecondMonitor =
```

Comment:
I assume we don't have to worry too much about the window being moved to another monitor after opening the popup?

Reply:
Actually I originally just put it in all the time, which works.  This current impl does have problems when going primary->secondary. However, I would rather not add extra CPU cost all the time for when you have window open, move monitors. if you reopen again, it'll be fine.

Reply:
Yeah, that sounds like a good compromise. Better prioritise good performance for 99.9% cases.